server/client.go

```diff
@@ -94,7 +94,7 @@ type client struct {
 	opts  clientOpts
 	start time.Time
 	nc    net.Conn
-	mpay  int
+	mpay  int64
```

Comment:
Move this to top of structure (say after `stats`) because of this: https://github.com/golang/go/issues/599